README.md

```diff
@@ -41,6 +41,7 @@ Conseil Config:
 
 
 * API_PORT - Conseil API port, default: 80
+* API_KEY - Conseil API key, default: conseil
```

Comment:
This is good for now but ideally we should take a plural of API keys. Perhaps we can do this under a separate ticket unless @Pierre-Michard feels generous enough to make the change now. :smile:

Reply:
Sorry, I can't think of an elegant way to configure an array of strings in an env variable.